vantage/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
```

Comment:
Hi team, thanks for opening up this PR! You can delete this file since you have nothing defined in `assets` in `manifest.json`